scripts/gitlab/build-windows.sh

```diff
@@ -14,6 +14,9 @@ echo "RUSTC_WRAPPER:    " $RUSTC_WRAPPER
 echo "SCCACHE_DIR:      " $SCCACHE_DIR
 
 echo "_____ Building target: "$CARGO_TARGET" _____"
+# NOTE: if you change this please remember to also update .cargo/config
+export RUSTFLAGS=" -Ctarget-feature=+aes,+sse2,+ssse3 -Ctarget-feature=+crt-static"
```

Comment:
Before this PR we had `+crt-static` only for the `x86_64-pc-windows-msvc` target, but I'm not sure we care about 32-bit windows?

Reply:
No, we do not care about 32-bit windows.